### PR TITLE
Python 3: remove autotest for libvirt_bench

### PR DESCRIPTION
**Pull request by hahakiki2010**
This PR depends on avocado-vt#1403

Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

**Comment by hahakiki2010** [comment]
depends on avocado-vt https://github.com/avocado-framework/avocado-vt/pull/1403

File: libvirt/tests/src/libvirt_bench/libvirt_bench_domstate_switch_with_unixbench.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def _is_unixbench_running():
     logging.debug("Unixbench is already running in VMs.")
 
     # Run unixbench on host.
-    from autotest.client import common
+    from virttest.utils_test import common
     autotest_client_dir = os.path.dirname(common.__file__)
     autotest_local_path = os.path.join(autotest_client_dir, "autotest-local")
```
Comment (balamuruhans) [inline libvirt/tests/src/libvirt_bench/libvirt_bench_domstate_switch_with_unixbench.py]: can we consider the naming convention as part of the port ?

Comment (hahakiki2010) [reply libvirt/tests/src/libvirt_bench/libvirt_bench_domstate_switch_with_unixbench.py]: @balamuruhans, this PR is targeted to let avocado list work well with no autotest. but, the script running still depends on autotest client "autotest-local", and need another PR for the replacement, you could see the comments in https://github.com/avocado-framework/avocado-vt/pull/1403

Comment (balamuruhans) [reply libvirt/tests/src/libvirt_bench/libvirt_bench_domstate_switch_with_unixbench.py]: Thanks!, got it

**Comment by iccaszhulili** [comment]
rh-caselink retest please

**Comment by kylazhang** [comment]
need to wait for depends pr merge first

**Comment by chunfuwen** [comment]
close it firstly since it depends on another long pending unmerged PR.

**Pull request closed without merging**